tools/donate-cpu-server.py

```diff
@@ -620,6 +620,8 @@ def timeReport(resultPath: str) -> str:
                     '{:.2f}'.format(time_factor).rjust(column_widths[3]) + '\n'
             break
 
+    html += '\n'
+    html += '(listed above are all suspicious timings with a factor <0.50 or >2.00)\n'
```

Comment:
maybe I miss something but I think you should use  `&lt;` and `&gt;` here.

Reply:
I guess you are right, this ought to be HTML code.